syntax = "proto3";
import "ris.proto";
package region.api;

message HandShake {
	uint32		version = 1;
}

message Result {
	int32		result = 1;
	uint32		version = 2;
	string		what = 3;
}

message AddService {
	ris.Service	svc		= 1;
	bool		rep		= 2;
}

message RmService {
	string		name		= 1;
	bool		rep			= 2;
}

message AddPayload {
	ris.Payload	payload		= 1;
	bool		rep			= 2;
}

message RmPayload {
	string		uuid		= 1;
	bool		rep			= 2;
}

